src/LibraryViewExtensionWebView2/LibraryViewController.cs

```diff
@@ -287,23 +287,23 @@ private string ReplaceUrlWithBase64Image(string html, string minifiedURL, bool m
         //list of resources which have paths embedded directly into the source.
         private readonly Tuple<string, bool>[] dynamicResourcePaths = new Tuple<string, bool>[]
         {
-           Tuple.Create("/resources/ArtifaktElement-Bold.woff",true),
```

Comment:
Why the "/" is not needed?

Reply:
> Why the "/" is not needed?

Because as we changed the public path in package so the bundle has change